packages/Webkul/Velocity/src/Resources/assets/sass/components/shared.scss

```diff
@@ -472,7 +472,6 @@
             float: left;
             width: 30%;
             height: 270px;
-            max-width: 200px;
```

Comment:
In the previous PR, you have removed height and in this one, you have removed width.